azure-stack/operator/compare-azure-azure-stack.md

```diff
@@ -55,7 +55,7 @@ In addition to the resource providers described above, there are additional PaaS
 
 ### Example use cases for Azure Stack Hub
 
-- Financial modeling
+- Financial modelling
 - Clinical and claims data
```

Comment:
This is the British spelling of the word, so the original spelling was correct.

Reply:
I applied the edit/reversal and will merge.